azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/AuthHelper.java

```diff
@@ -85,17 +94,41 @@ public static AzureEnvironment parseAzureEnvironment(String environment) {
             return null;
         }
 
-        switch (StringUtils.remove(environment.toLowerCase(Locale.ENGLISH), "_")) {
+        switch (preprocessAzureEnvironment(environment)) {
```

Comment:
Change switch to `List.contains`? as there is no difference in valid azure environments

Reply:
fix in new PR